dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -28,15 +28,20 @@
                   %span.puzzle-number
                     = @script_level.position
               - if @user_level.submitted?
-                .submitted
-                  %h4= t('teacher.panel.submitted_on', date: @user_level.updated_at)
+                #submitted
+                  %div= t('teacher.panel.submitted_on')
+                  .timestamp= @user_level.updated_at
                   %button.btn.btn-info#unsubmit{"data-user-level-url" => url_for(@user_level)}
                     = t('teacher.panel.unsubmit')
                   :javascript
+                    $(function() {
+                      var submittedTimestamp = $('#submitted .timestamp');
+                      submittedTimestamp.text((new Date(submittedTimestamp.text())).toLocaleString());
```

Comment:
I tried it on IE9 on saucelabs and it does have toLocaleString.  According to do the doc IE<11 doesn't support toLocaleString() with the locale as an argument. The doc was kind of unclear about whether it supports it without an argument as I'm using it here but I tried it and it seems to:

<img src="https://www.dropbox.com/s/j7i1ik8emqj1xcq/Screenshot%202015-10-20%2013.31.40.png?dl=1">

However, IE9 doesn't like to parse the output of Time#to_s, fixing...